app/models/search.rb

```diff
@@ -128,6 +129,11 @@ def filter_single(param, field)
   # https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html
   def aggregations
     {
+      collections: {
```

Comment:
FYI: I checked this code against timex-prod and confirmed it doesn't break if we release this before we Elasticsearch has the updated model. It just returns an empty array here like:
``` 
"collection": [],
```